setup.py

```diff
@@ -12,6 +12,7 @@
     "matplotlib==3.5.2",
     "numpy==1.21.6",
     "openai==0.23.0",
+    "litellm==0.13.2",
```

Comment:
That's not the right place for the requirement as this is for DS1000 task that doesn't support external APIs, it seems `openai` package wasn't properly added.

I would leave this list as it is and create a new variable 
`external_api_requirements=["litellm==0.13.2"]` then  add it  to the `extras_require` dict [here](https://github.com/bigcode-project/bigcode-evaluation-harness/blob/9d17482e38eaeddee5e27488229a893dc59d42e1/setup.py#L36) so people that want this evaluation install the package separately